src/pulse3D/excel_writer.py

```diff
@@ -359,7 +363,9 @@ def write_xlsx(
         min_value = min(interpolated_force)
         interpolated_force -= min_value
         interpolated_force *= MICRO_TO_BASE_CONVERSION
-
+        #find the biggest activation twitch force over all
+        if max(interpolated_force) > max_activeTwitchForce:
```

Comment:
A simple pattern for tracking a max value is `max_val = max(max_val, new_val)`. You can make this simpler and more clear by removing the if branch and just doing
```
max_force_of_well = max(interpolated_force)
max_force_of_recording = max(max_force_of_recording, max_force_of_well)
```